packages/augur-tools/src/flash/scripts.ts

```diff
@@ -344,9 +351,12 @@ export function addScripts(flash: FlashSession) {
       if (extraStake) {
         preEmptiveStake = new BigNumber(extraStake).dividedBy(QUINTILLION).toFixed();
       }
+      if (!this.usingSdk) this.log("This script needs sdk, make sure to connect with -u flag");
```

Comment:
This doesn't leave this function so it's going to error later when trying to actually do things.